packages/cli/src/Version.ts

```diff
@@ -143,6 +166,39 @@ export class Version implements Command {
     return []
   }
 
+  private async getPrismaClientJSGenerators(schemaPath: string | null): Promise<
```

Comment:
This function should take the `{ schemas }` output of `getSchemaPath` as input, rather than calling `getSchema()` internally.